templates/admin/lbf/edit_lbf/partials/modify_group.html.twig

```diff
@@ -0,0 +1,37 @@
+<!-- template DIV that appears when user chooses to add a new group -->
+<div class="modal" id="{{ id }}group" tabindex="-1" aria-labelledby="{{ id }}GroupModal" aria-hidden="false">
```

Comment:
Would pipe all the id outputs to attr in this block

Reply:
I don't think we need to escape in this context, we are in total control of the variable and it's either "new" or "rename"

Reply:
could just think about it this way.
It will do no harm (will not break any html) placing it there, will only be a couple keystrokes, and then never, ever need to worry about it again (maybe in 2 years somebody decides to allow some sort of user input to get into id and we are not around to remember this).